forge-game/src/main/java/forge/game/GameAction.java

```diff
@@ -481,33 +480,10 @@ private Card changeZone(final Zone zoneFrom, Zone zoneTo, final Card c, Integer
 
             // 400.7a Effects from static abilities that give a permanent spell on the stack an ability
             // that allows it to be cast for an alternative cost continue to apply to the permanent that spell becomes.
-            if (zoneFrom.is(ZoneType.Stack) && toBattlefield) {
-                List<KeywordInterface> newKw = Lists.newArrayList();
-                for (Table.Cell<Long, Long, KeywordsChange> cell : c.getChangedCardKeywords().cellSet()) {
-                    // comes from a static ability
-                    if (cell.getColumnKey() == 0) {
-                        continue;
-                    }
-                    for (KeywordInterface ki : cell.getValue().getKeywords()) {
-                        boolean keepKeyword = false;
-                        for (SpellAbility sa : ki.getAbilities()) {
-                            if (!sa.isSpell()) {
-                                continue;
-                            }
-                            if (sa.getAlternativeCost() != null) {
-                                keepKeyword = true;
-                                break;
-                            }
-                        }
-                        if (keepKeyword) {
-                            ki.setHostCard(copied);
-                            newKw.add(ki);
-                        }
-                    }
-                }
-                if (!newKw.isEmpty()) {
-                    copied.addChangedCardKeywordsInternal(newKw, null, false, copied.getTimestamp(), 0, true);
-                }
+            if (zoneFrom.is(ZoneType.Stack) && toBattlefield && c.getCastSA() != null && c.getCastSA().getKeyword() != null) {
```

Comment:
It gets messed up somehow and will have Blitz twice :/

![grafik](https://user-images.githubusercontent.com/8506892/220839518-22403028-d4b4-4c3d-aa29-c2e828693a4e.png)

Reply:
i make it only for non intrinsic ones (that should be Henzi)